mods/raclassic/rules/infantry.yaml

```diff
@@ -417,6 +417,7 @@ THF:
 		StandSequences: stand
 	Buildable:
 		Description: Steals enemy credits.\n  Unarmed
+		Prerequisites: ~tent, atek
```

Comment:
I have done this more properly (with other Buildable: vaules) with my tech level changes that i'll be filing a PR in a short while, i think you can drop this.